src/scripts/main.js

```diff
@@ -1,3 +1,59 @@
 'use strict';
 
+const table = document.querySelector('.field');
+const tbody = table.querySelector('tbody');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+
+appendRow.addEventListener('click', () => {
+  const createRow = document.createElement('tr');
+  const columnCount = table.rows[0].cells.length;
+
+  if (tbody.rows.length >= 10) {
+    appendRow.setAttribute('disabled', 'true');
+  }
+
+  for (let i = 0; i < columnCount; i++) {
+    const td = document.createElement('td');
+
+    createRow.append(td);
+  }
+  tbody.appendChild(createRow);
+});
+
+removeRow.addEventListener('click', () => {
+  if (tbody.rows.length > 2) {
+    tbody.deleteRow(tbody.rows.length - 1);
+  }
+
+  if (tbody.rows.length <= 2) {
+    removeRow.setAttribute('disabled', 'true');
+  }
```

Comment:
The logic for disabling the 'removeRow' button is correct, but there is no logic to re-enable it when the number of rows is more than 2. Consider adding logic to enable the button when rows are added and the count is above 2.